grc/core/FlowGraph.py

```diff
@@ -223,8 +223,11 @@ def renew_namespace(self):
         # Load variables
         for variable_block in self.get_variables():
             try:
+                variable_block.rewrite()
```

Comment:
What made the variable_block need the rewrite at this point?

Reply:
Otherwise variable_block.namespace is 'invalid' and the following eval fails.
After the rewrite the variable_block.namespace is valid and the eval works as expected.

block.namespace is defined as
`    def namespace(self):
        return {key: param.get_evaluated() for key, param in six.iteritems(self.params)}
`
Without the rewrite param.get_evaluated() returns None